app/views/wigs/show.html.erb

```diff
@@ -5,10 +5,10 @@
     <div class="vertical-flex">
       <div class="main-description">
         <h1>Rent <%= @wig.title %></h1>
-        <p class="description">Description:</p><%= @wig.description %></p>
-        <p class="description">Color:</p> <p> <%= @wig.color %></p>
-        <p class="description">Size:</p><%= @wig.size %></p>
-        <p class="description">Price:</p><%= @wig.price%>€ </strong> per day</p>
```

Comment:
Lets keep the old layout with line breaks please! We need a real way to fix it because otherwise it will break on another reolution again